administration/logging.md

```diff
@@ -127,89 +131,49 @@ log:set DEBUG org.openhab.binding.zwave
+2016-06-04 16:28:39.482 [DEBUG] [org.openhab.core.model.script.heating] Bedroom: Temperature 21.3°C, Mode NORMAL
 ```
 
 Note that, in the last example above, inclusion and formatting of values is done using [Java Formatter String Syntax](https://docs.oracle.com/javase/7/docs/api/java/util/Formatter.html).
 
-## Logging into Separate File
+## Logback configuration and logging into separate files
 
-Per default all log entries are saved in the file `openhab.log` and event specific entries are saved in `events.log`.
+As per default, all log entries are saved in the file `openhab.log` and event specific entries are saved in `events.log`.
 Additional files can be defined in order to write specifics logs to a separate place.
 
-In order to create a new log file following two areas needs to be added to the [configuration file](#config-file):
-
-**New logger:**
-
-```java
-# ZWave Logger
-log4j2.logger.ZWave.name = org.openhab.binding.zwave
-log4j2.logger.ZWave.level = INFO
-log4j2.logger.ZWave.additivity = false
-log4j2.logger.ZWave.appenderRefs = ZWave
-log4j2.logger.ZWave.appenderRef.ZWave.ref = ZWAVE
-```
-
-**New file appender:**
-
-```java
-# ZWave File Appender - ZWave.log
-log4j2.appender.ZWave.name = ZWAVE
-log4j2.appender.ZWave.type = RollingRandomAccessFile
-log4j2.appender.ZWave.fileName = ${openhab.logdir}/ZWave.log
-log4j2.appender.ZWave.filePattern = ${openhab.logdir}/ZWave.log.%i
-log4j2.appender.ZWave.immediateFlush = true
-log4j2.appender.ZWave.append = true
-log4j2.appender.ZWave.layout.type = PatternLayout
-log4j2.appender.ZWave.layout.pattern = %d{yyyy-MM-dd HH:mm:ss.SSS} [%-5.5p] [%-36.36c] - %m%n
-log4j2.appender.ZWave.policies.type = Policies
-log4j2.appender.ZWave.policies.size.type = SizeBasedTriggeringPolicy
-log4j2.appender.ZWave.policies.size.size = 10MB
-log4j2.appender.ZWave.strategy.type = DefaultRolloverStrategy
-log4j2.appender.ZWave.strategy.max = 10
-```
-
-## Logback Configuration File
-
-In order to define custom log patterns, log to network sockets and so on we can prepare a logging configuration file.
-
-There are several things, that you might want to change in the configuration:
+In order to define custom log patterns, log to network sockets and so on we can prepare a logging configuration in `log4j2.xml`.
+There are several things that you might want to change in the configuration:
 
 - the log level for a logger;
 - the pattern of an appender;
 - redirect the log to a text file.
 
-The configuration file for openHAB is placed in the [openhab-distro/lauch/home/logback_debug.xml](https://github.com/openhab/openhab-distro/blob/master/launch/home/logback_debug.xml) file. We have added a few comments on this file in order to attract your attention on some significant points:
+<!--
+The configuration file for openHAB is placed in the [openhab-distro/lauch/home/logback_debug.xml](https://github.com/openhab/openhab-distro/blob/master/launch/home/logback_debug.xml) file. 
```

Comment:
Logback is only used during development (which isn't very clear from the current docs).

The logback config file is now: https://github.com/openhab/openhab-distro/blob/master/launch/app/runtime/logback.xml

It's XML configuration format is very similar to Log4j, see also:

* https://logging.apache.org/log4j/2.x/manual/configuration.html
* https://logback.qos.ch/manual/configuration.html

So for Log4j we probably want a similar section but with the Log4j XML syntax see: 

https://github.com/openhab/openhab-distro/blob/master/distributions/openhab/src/main/resources/userdata/etc/log4j2.xml

It might be an idea to move the Logback configuration details to a separate page because it might become confusing for normal users. They will likely start using Logback configuration syntax in Log4j files. :neutral_face:

Reply:
So if logback is not in use by users (just devs as you say) it's probably safest to delete all of this from the docs.
I guess you yourself don't need this page any more, do you ;-) ?
So will delete what I commented out.

Reply:
>It might be an idea to move the Logback configuration details to a separate page because it might become confusing for normal users. They will likely start using Logback configuration syntax in Log4j files. 😐

Maybe the development area would be a good place for this.